### PR TITLE
PE-2386: fix download files on web

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4dfliaplq13s0
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/50a7em800d6d0

### CODE REVIEW

**Review by matibat — DISMISSED** [review]
LGTM

**Review by karlprieb — APPROVED** [review]

File: lib/blocs/upload/upload_handles/file_data_item_upload_handle.dart [hunk 1]
```diff
@@ -72,15 +69,8 @@ class FileDataItemUploadHandle implements UploadHandle, DataItemHandle {
...
-    late Uint8List fileData;
-    if (File(file.ioFile.path).existsSync()) {
-      fileData = await file.ioFile.readAsBytes();
-    } else {
-      debugPrint('getting file from local storage');
-      final cache = IOCacheStorage();
-      final cachedFile = await cache.getFileFromStorage(file.ioFile.name);
-      fileData = await cachedFile.readAsBytes();
-    }
+    final fileData = await file.ioFile.readAsBytes();
```
Comment (thiagocarvalhodev) [inline lib/blocs/upload/upload_handles/file_data_item_upload_handle.dart:72]: This is already cached (on mobile, specifically on iOS).

Comment (thiagocarvalhodev) [reply lib/blocs/upload/upload_handles/file_data_item_upload_handle.dart:72]: https://github.com/ardriveapp/ardrive-web/pull/714/files#r986866800

**Review by karlprieb — APPROVED** [review]

**Review by matibat — APPROVED** [review]
LGTM

**Pull request merged**